internal/pkg/mapper/defaultbucket/local_load.go

```diff
@@ -0,0 +1,59 @@
+package defaultbucket
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/orderedmap"
+)
+
+// MapAfterLocalLoad - replace placeholders with default buckets in IM.
+func (m *defaultBucketMapper) MapAfterLocalLoad(recipe *model.LocalLoadRecipe) error {
+	config, ok := recipe.Object.(*model.Config)
+	if !ok {
+		return nil
+	}
+
+	err := m.visitStorageInputTables(config, m.replacePlaceholderWithDefaultBucket)
+	if err != nil {
+		m.Logger.Warnf(`Warning: %s`, err)
+	}
+	return nil
+}
+
+func (m *defaultBucketMapper) replacePlaceholderWithDefaultBucket(config *model.Config, inputTableSource string, inputTable *orderedmap.OrderedMap) error {
+	if !strings.HasPrefix(inputTableSource, "{{:default-bucket:") {
+		return nil
+	}
+
+	sourceWithoutPrefix := strings.TrimPrefix(inputTableSource, "{{:default-bucket:")
+	splitSource := strings.Split(sourceWithoutPrefix, "}}.")
+	if len(splitSource) != 2 {
+		return nil
+	}
+
+	// Get branch
+	branch := m.State.MustGet(config.BranchKey())
+
+	// Get key by path
+	path := filesystem.Join(branch.Path(), splitSource[0])
+	configKeyRaw, found := m.Naming.FindByPath(path)
```

Comment:
Si v `MapAfterLocalLoad` a tu nie je garantovane, ze ten druhy config je uz nacitany :/
Vid poznamku v `type LocalLoadMapper`.

Je potrebne pouzit `OnLocalChange(changes *model.LocalChanges)` a `changes.Loaded()`.

Reply:
Note: Ak mas nejaky napad na lepsi nazov pre `OnLocalChange`, tak to kludne premenuj. Su tam `Reamed, Deleted` ... ale aj `Loaded`, tj. vsetko co spravil `local.UnitOfWork`. Lepsi nazov mi nenapadol.

Reply:
Unit test samozrejme presiel, ale funckcionalny by nahodne padal. Podla toho, ktory config sa podari nacitat skor.

Reply:
Ok, fixnuto.